test/helpers/remote_execution_helper_test.rb

```diff
@@ -23,4 +23,20 @@ class RemoteExecutionHelperTest < ActionView::TestCase
                                                         "Exit status: 0"])
     end
   end
+
+  describe 'test correct setting' do
+    it 'should found correct template from setting' do
+      Setting::RemoteExecution.load_defaults
+      template_name = 'Job Invocation Report Template'
+      setting_key = 'remote_execution_job_invocation_report_template'
+      template = FactoryBot.create(:report_template, name: template_name)
+      input = FactoryBot.create(:template_input, name: 'job_id', input_type: 'user')
+      template.template_inputs << input
+      Setting[setting_key] = template_name
```

Comment:
We would be bit better of with, but not a big deal.
```suggestion
      Setting.expects(:[]).with(setting_key).returns(template_name)
```